packages/form-js-viewer/test/spec/Form.spec.js

```diff
@@ -288,6 +288,32 @@ describe('Form', function() {
   });
 
 
+  it('should render empty', async function() {
```

Comment:
I'd love to have an integration test to verify the behavior (`null` fields being submitted). @philippfromme

Reply:
From what I see we test unit-test wise only, not end-to-end at the moment.

Reply:
I've added that test in a follow-up pull request: https://github.com/bpmn-io/form-js/pull/210/files#diff-f7f5270dd58d99d4b13f56770a7bcd44bee6b5d7a2f7d467fc4cf29b8f6d5093R314 🤡 Would that be alright?

Reply:
Very nice :).